modules/fargate/main.tf

```diff
@@ -63,5 +63,10 @@ resource "aws_eks_fargate_profile" "this" {
     }
   }
 
+  timeouts {
+    create = lookup(each.value["timeouts"], "create", 10)
```

Comment:
instead assuming some values put failover to `null` value which will use provider defaults

Reply:
```suggestion
    create = lookup(each.value["timeouts"], "create", null)
```